.github/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+
+registries:
+  npm-github:
```

Comment:
Hm, skal ikke dette helst inn  i .npmrc fila?

Reply:
Ikke helt sikker på hva du mener, men tenker du at det burde ligge i .npmrc-fila som ligger i rota på prosjektet?
Den peker jo på en env variabel som jeg har lokalt $NPM_AUTH_TOKEN, som vel er min personlige token. Mens READER_TOKEN er en token som er lagt inn som secret i github settings spesielt for dependabot. Er forøvrig kopiert fra oppsettet til flex.

Reply:
Her er doken:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#registries